app/scripts/modules/core/src/widgets/spinners/Spinner.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+
+const Logo = require('./logo.svg');
+
+export interface ISpinnerProps {
+  size?: string;
+  message?: string;
+  postnote?: string;
+}
+
+export class Spinner extends React.Component<ISpinnerProps, {}> {
+  public static propTypes = {
+    size: React.PropTypes.oneOf(['nano', 'small', 'medium', 'large', 'page']),
+    message: React.PropTypes.string,
+    postnote: React.PropTypes.string
+  }
+
+  public getBarRows = (count: number) => {
+    const rows = [];
+    let i: number;
+    for (i = 0; i < count; i++) {
+      rows.push(<div className="bar" />)
+    }
+    return rows;
+  }
+
+  public renderNano = () => {
+    return (
+      <div className="load nano">
+        {this.getBarRows(1).map(bar => bar)}
```

Comment:
I kept it for consistency since the function returns an array. Removed it for this single one.